### ISSUE TITLE
ICANN pipemail test failing

### ISSUE THREAD

**Issue by sbenthall** [datactive/bigbang#608]
Mysterious failures of the ICANN pipermail tests...

https://gist.github.com/sbenthall/7c59aa9755140120614b80cd9aaf521e

Some combination of SSL certification failures, exceeded retries, and Nonetypes where they shouldn't be.

`urllib3.exceptions.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1129)`

`urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='mm.icann.org', port=443): Max retries exceeded with url: /pipermail/accred-model/2018-August.txt.gz (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1129)')))`

`AttributeError: 'NoneType' object has no attribute 'select'`

Basically, retrieving the mail isn't working.
CodeCov was firing errors for these tests and that's messing with other development workflow.
So I'll disable these tests in the short term until we can figure out how to fix these issues.
